code/modules/admin/secrets/fun_secrets/change_credits.dm

```diff
@@ -7,7 +7,7 @@ var/global/end_credits_title
 	name = "Change End Credits Title"
 
 /datum/admin_secret_item/fun_secret/change_credits_song/do_execute()
-	var/list/sounds = subtypesof(/decl/music_track)	
+	var/list/sounds = decls_repository.get_decls_of_subtype(/decl/music_track)	
```

Comment:
Either change line 15 to not use `GET_DECL()` or change this to `get_decl_paths_of_subtype()`.